admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -32,17 +32,19 @@ of memcache that best fits your needs. The supported caching backends are:
    A local cache for systems running PHP 5.5 and up, APCu 4.06 or better required.
 * `Memcached <http://www.memcached.org/>`_ 
    Distributed cache for multi-server ownCloud installations.
-* `Redis <http://redis.io/>`_ 
+* `Redis <http://redis.io/>`_, PHP module 2.2.5 or better required.
    For distributed caching, and required for Transactional File Locking.
 
 These are supported but not recommended:
 
-* `XCache <http://xcache.lighttpd.net/>`_ 
+* `XCache <http://xcache.lighttpd.net/>`_ (might need some special configuration
+  not described in this documentation)
```

Comment:
I suggest leaving out the XCache comment, as we don't want to encourage using it in any way. AFAIK it may not work at all; I'm still trying to get a definitive answer.